### PR TITLE
[IMP] mail: appropriate padding around chatbox search bar

### PR DESCRIPTION
**Pull request by ibo-odoo**
Set appropriate padding around the chatbox search bar and aligned it with the
header content.

**Task**-2844934

### CODE REVIEW

**Comment by ibo-odoo** [comment]
@seb-odoo A small IMP

File: addons/mail/static/src/components/chat_window/chat_window.xml [hunk 1]
```diff
@@ -31,7 +31,7 @@
                     />
                 </t>
                 <t t-if="chatWindow.newMessageAutocompleteInputView">
-                    <div class="o_ChatWindow_newMessageForm d-flex align-items-center mt-1 p-1">
+                    <div class="o_ChatWindow_newMessageForm d-flex align-items-center p-3">
```
Comment (seb-odoo) [inline addons/mail/static/src/components/chat_window/chat_window.xml:34]: Usually prefer margin than padding when both work. Padding should typically be used when the inside size matters (eg. if there is a visible border or different background color).

**Comment by ibo-odoo** [comment]
@seb-odoo Done.

**Review by seb-odoo — APPROVED** [review]
@robodoo r+

**Pull request closed without merging**